picasso/src/main/java/com/squareup/picasso/PicassoDrawable.java

```diff
@@ -153,13 +153,13 @@ private void drawDebugIndicator(Canvas canvas) {
   }
 
   private static Path getTrianglePath(Point p1, int width) {
```

Comment:
You can amend the commit and force push your branch. It will automatically update this pull request.